api/client/swarm/inspect.go

```diff
@@ -19,7 +19,7 @@ func newInspectCommand(dockerCli *client.DockerCli) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "inspect [OPTIONS]",
-		Short: "Inspect the Swarm",
+		Short: "Inspect the Swarm.",
```

Comment:
I think dots should not be there at all (most of the commands don't have dots either)